src/components/ui/Popover.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import { cn } from '@/lib/tailwind/utils';
+
+import {
+  Content as PrimitiveContent,
+  Portal as PrimitivePortal,
+  Root as PrimitiveRoot,
+  Trigger as PrimitiveTrigger,
+} from '@radix-ui/react-popover';
+import { ComponentPropsWithoutRef, ElementRef, forwardRef } from 'react';
+
+const Popover = PrimitiveRoot;
+const PopoverTrigger = PrimitiveTrigger;
+
+const PopoverContent = forwardRef<
+  ElementRef<typeof PrimitiveContent>,
+  ComponentPropsWithoutRef<typeof PrimitiveContent>
+>(({ className, align = 'center', sideOffset = 4, ...props }, ref) => (
+  <PrimitivePortal>
+    <PrimitiveContent
+      ref={ref}
+      align={align}
+      sideOffset={sideOffset}
+      className={cn(
+        'z-50 w-[calc(100vw-1rem)] max-h-[calc(100vh_/_2_-_var(--radix-popover-trigger-height))]',
```

Comment:
It seems like working perfect, but not sure why we don't need `_` in `w-[calc(100vw-1rem)]`, like `w-[calc(100vw_-_1rem)]`... I just don't understand it enough maybe

Reply:
I understand that an underscore is not required unless you want to recreate a space. In general calculations, you can omit spaces instead of replacing them with underscores.

https://tailwindcss.com/docs/content#using-spaces-and-underscores